portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,77 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+ *{
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+    } 
+    
+#resume-header {
+    margin-top: 20px;
+    margin-bottom: 20px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#profile-picture {
+    width: 100%;
 }
+
+#resume-footer {
+    padding: 20px;
+    text-align: center;
+}
+
+.main-nav
```

Comment:
Based on the [CSS style guide](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation), for this selector and the selectors below, it is preferred to have the opening brace `{` be on the same line as the selector. For example, instead of
```
.main-nav
{
```
it would be `.main-nav {`.